src/System.Runtime.Loader/ref/System.Runtime.Loader.netcoreapp.cs

```diff
@@ -0,0 +1,16 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+// ------------------------------------------------------------------------------
+// Changes to this file must follow the http://aka.ms/api-review process.
+// ------------------------------------------------------------------------------
+
+namespace System.Runtime.Loader
+{
+    public sealed class AssemblyDependencyResolver
+    {
+        public AssemblyDependencyResolver(string componentAssemblyPath) { }
+        public string ResolveAssemblyToPath(System.Reflection.AssemblyName assemblyName) { throw null; }
+        public string ResolveUnmanagedDllToPath(string unmanagedDllName) { throw null; }
```

Comment:
"unmanaged dll"... is this Windows only? If yes, why? If not, I assume it would work for "so" on Linux, "dylib" on macOS, etc., so why have "dll" in the name?

Reply:
"dll" is in the name so that the name is similar to the method that would most likely call it, an overridden `System.Runtime.Loader.AssemblyLoadContext.LoadUnmanagedDll` method.

Reply:
I see. Seems like unfortunate naming on ALC, but I see the logic in being consistent with it. Ok.